e2e/components/Dialog/DialogWrapper.test.ts

```diff
@@ -10,7 +10,7 @@ test('DialogContent が開閉できること', async (t) => {
   const trigger = Selector('[data-test=dialog-trigger]')
   const content = Selector('[data-test=dialog-content]')
   const closer = Selector('[data-test=dialog-closer]')
-  const background = content.parent('[role=dialog]').find('.smarthr-ui-Dialog-background')
+  const background = Selector('#dialog-uncontrolled').find('.smarthr-ui-Dialog-background')
```

Comment:
[IMO] できれば e2e テストでは id による要素の参照は避けたいという考えが少しあります。 id 参照していると Story 側の id を変更するとテストが通らなくなりますが、そのことは Story 編集時点では判断できないためです。
それを避けるために `data-test` によってテストのための値であることを明確にしていて、今回の場合はこのように `data-test` 参照で記述できると思います。

```suggestion
  const background = content
    .parent('.smarthr-ui-Dialog-wrapper')
    .find('.smarthr-ui-Dialog-background')
```

Reply:
@wmoai こっちも対応してます！